nextflow.config

```diff
@@ -12,10 +12,18 @@ params {
     processed_acc         = null
     external_input        = null
     external_metadata     = null
+    cleanup               = true
+    debug                 = false
     output_path           = "$PWD/results"
 
+    max_memory            = '100.GB'
+    max_cpus              = 1
+    max_time              = '25.h'
```

Comment:
Same, don't limit the maxes

Reply:
Ok, I've changes this to the following as in the most Mgnify pipelines:
```
    max_memory        = '128.GB'
    max_cpus              = 32
    max_time               = '240.h'
```